tests/utils/exec.ts

```diff
@@ -629,171 +604,33 @@ async function setupTestAgent(privateKeyPem: PrivateKeyPem, logger: Logger) {
 }
 
 function spawnFile(path: string) {
-  return child_process.spawn('ts-node', [
+  return childProcess.spawn('ts-node', [
     '--require',
     'tsconfig-paths/register',
     path,
```

Comment:
This is no longer necessary because it is in the `tsconfig.json`. Can you make sure that the nodes tests are also no longer doing this. I don't think @tegefaulkes fixed that up in the last PR.

Reply:
It seems that only the `types` are used now in nodes, so this is OK. But it should be better to do `import type { ... } from 'child_process';` instead.

```
nodes/NodeConnection.test.ts
5:import type * as child_process from 'child_process';
736:      let testProcess: child_process.ChildProcessWithoutNullStreams | undefined;
802:      let testProcess: child_process.ChildProcessWithoutNullStreams | undefined;
```

Reply:
Use `import type { ... } from 'child_process';` here instead. That way no confusion with using childProcess directly.